jena-base/src/main/java/org/apache/jena/atlas/lib/CacheFactory.java

```diff
@@ -33,7 +33,7 @@
 
     /** Create a null cache */
     public static <Key, Value> Cache<Key, Value> createNullCache() {
-        return new Cache0<>() ;
+        return new CacheGuava<>(0) ;
     }
```

Comment:
Cache0 is really just a "complete the set" implementation.  If any use as a dummy, the weight of CacheGuava might not be worth it.  Proposal: just remove Cache0 (it does not fit with getOrFill anyway).